Dockerfile

```diff
@@ -35,6 +35,9 @@ RUN echo "if [ -d \"\$HOME/.local/bin\" ] ; then\nPATH=\"\$HOME/.local/bin:\$PAT
 # Set the buildlogs directory into /tmp as other directory aren't accessible
 ENV BUILDLOGS=/tmp/buildlogs
 
+RUN sudo apt-get update && sudo apt-get install -y gcc-arm-none-eabi
```

Comment:
this is wrong ... we already have the right toolchain in the container and don't want to use Ubuntu one's

Reply:
It definitely wouldn't build for me without this, maybe there were path issues? It wouldn't find the compiler.

Reply:
You can try removing that line and running the script to reproduce.

Reply:
Just enter the container and you will be able to compile fine

Reply:
Hmm, so it is a path issue? I wanted to be able to run one command and get the output, otherwise it's too many steps. I can look into adding the proper path additions to compile, though.

Reply:
Well you are correct, running `bash` and then `./waf build` inside the container works fine... I've reverted this line and will investigate further.